editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx

```diff
@@ -65,6 +65,11 @@ export const FileTaggingModal = ({
           borderTop: (theme) => `20px solid ${theme.palette.primary.main}`,
           background: "#FFF",
           margin: (theme) => theme.spacing(2),
+          height: "100%",
+          "@media screen and (max-width: 500px)": {
```

Comment:
```suggestion
          [theme.breakpoints.down("md"): {
```

I _think_ this is the right MUI syntax ([docs](https://mui.com/material-ui/customization/breakpoints/#css-media-queries)) - please test this though and don't trust me blindly! 😂

Reply:
I did try that but was given the error `Cannot find name 'theme'`. Additionally I couldn't find another instance of using theme breakpoints within `sx` styling.

Reply:
In order to access `theme`, you'll need to change `sx` to be a function, e.g.

```tsx
PaperProps={{
  sx: (theme) => ({
     ...
     [theme.breakpoints.down("md")]: {
       margin: 0,
       maxHeight: "none",
     },
   })
 })
 ```

Reply:
Docs here on this: https://mui.com/system/getting-started/the-sx-prop/#callback-values

Reply:
Thanks for this, updated commit incoming